docker-compose.override.yml

```diff
@@ -10,7 +10,7 @@ version: '3.7'
 services:
   #db:
   web:
-    image: ghcr.io/canastawiki/canasta:1.3.0
+    image: canastafarm
```

Comment:
Make sure this is removed upon pushing to prod

Reply:
Sure, I will set https as default and if user set url start with http:// then set it to http://